apps/mobile/src/components/DrawerNavigation/Sidebar.tsx

```diff
@@ -0,0 +1,53 @@
+import type { FC } from 'react';
+import { View } from 'react-native';
+import type { DrawerContentComponentProps } from '@react-navigation/drawer';
+import DashboardNavigator from '@walless/app/features/DashboardLayout/Navigator';
+import {
+	useSafeAreaInsets,
+	useSnapshot,
+	useWidgets,
+} from '@walless/app/utils/hooks';
+import { appState, mockWidgets, widgetActions } from '@walless/engine';
+import type { WidgetDocument } from '@walless/store';
+import { appActions } from 'state/app';
+
+export const sidebarWidth = 64;
+
+const IS_HARDCODED = true;
```

Comment:
we may not need this flag, just directly use `mockWidgets` if we did not have real one -- for local debug go change the `widgets` prop directly.

Reply:
I've replied in the below conversation

Reply:
as discussed offline, still need to remove `IS_HARDCODED` flag